src/main/java/com/andmcadams/wikisync/WikiSyncPlugin.java

```diff
@@ -269,6 +270,14 @@ private void merge(PlayerData oldPlayerData, PlayerData delta)
 
 	private void submitPlayerData(PlayerProfile profileKey, PlayerData delta, PlayerData old)
 	{
+		// If cyclesSinceSuccessfulCall is not a perfect square, we should not try to submit.
+		// This gives us quadratic backoff.
+		cyclesSinceSuccessfulCall += 1;
+		if (Math.pow((int) Math.sqrt(cyclesSinceSuccessfulCall), 2) != cyclesSinceSuccessfulCall)
```

Comment:
Not sure if theres a less silly way to do this, I didnt see an obvious one. Just `(sqrt(x))^2 == x`